### PR TITLE
Makes the Mjolnir even more legendary

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Makes the Mjolnir more like its legends, the wielder of the Mjollnir gains immeasurable strength.

 - When dropped the Mjolnir will smash into the ground and anchor itself, being unable to be picked up by non-wizards.
 - When thrown the Mjolnir will accelerate out of the wielders hands and towards targets. It will smash through windows and structures on the way. If it hits a wizard, the wizard will catch it, otherwise it will send the mob flying.

Got bored while watching a wizard and always wanted to make the Mjolnir unable to be picked up by other people

## Why It's Good For The Game

I've always thought the Mjolnir was lacking compared the legends, how comes non-wizards can pick this up? The Mjolnir is pretty cool in that they thought about anti-powergaming when they came up with the legend by making it so greytiders are unable to lift it.

## Testing Photographs and Procedure

image

image

image

image

image

image

https://user-images.githubusercontent.com/26465327/205747820-9e018977-6e41-4c86-a97c-5cb51286f2b7.mp4

## Changelog
:cl:
balance: Mjolnir can only be picked up by wizards.
balance: Mjolnir can be thrown with immeasurable strength.
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
amazing. Very nice work.

Could do with a resprite though, its looking a bit dated

**Comment by PowerfulBacon** [comment]
I kinda wanna buff the singularity hammer too but I got no ideas

**Comment by Ivniinvi** [comment]
inconsistent spelling - mjolnir vs mjollnir

**Comment by EvilDragonfiend** [comment]
The real spelling Mjölnir is cooler

**Comment by Fronsis** [comment]
Hmm, It might be me but the level of screen shake might be a little too much? Bah, i can imagine a fight being a little hard on the eyes, I imagine a Wiz constantly throwing-recalling vs a couple of security members and it would result in a very.. non pleasant spam of shake as if you were bombarded by mini-meteors(not to add that the wiz also has some other explosive spells that are too visual) In fact just grab+q+grab+q would already be an annoying spam(imagine the first five seconds of the video but spammed)

I get that you're trying to portray the might and legendary status of the hammer, the sound is already good i'm just not too fond of the shake.

Also i agree that the sprite certainly didn't age well

**Review by itsmeow — DISMISSED** [review]
rename all instances of mjollnir to mjolnir please

File: code/game/objects/items/mjolnir.dm [hunk 1]
```diff
@@ -0,0 +1,170 @@
+/obj/item/mjolnir
+	name = "Mjolnir"
+	desc = "A weapon worthy of a god, able to strike with the force of a lightning bolt. It crackles with barely contained energy."
+	icon_state = "mjollnir0"
```
Comment (PestoVerde322) [inline code/game/objects/items/mjolnir.dm:4]: Fix the icon state's name here to avoid typos and whatnot.

Also other instances too.

**Review by PestoVerde322 — APPROVED** [review]
Approved, will fix icon name post-merge.

**Pull request merged**